CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-07-02
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). 
+* Starting Kubernetes 1.25, the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+* A [known issue](https://github.com/Azure/AKS/issues/3718) in Kubernetes version 1.24 is causing name resolution failures in Windows pods. Customers experiencing this issue should upgrade their cluster to Kubernetes version 1.25.
+* Learn [how to deploy an application that uses OpenAI on AKS](https://learn.microsoft.com/en-us/azure/aks/open-ai-quickstart?tabs=aoai) by running a sample multi-container solution for an e-commerce website and deploying OpenAI service to generate product descriptions.
+
+### Release notes
+
+* Preview Features
+  * Added Prometheus scraping annotations to App Routing add-on.
```

Comment:
Prometheus scraping annotations are means to the end. We need to reword this to convey what the user gets as benefit... For example, metrics under prometheus on nginx used in WAR addon. Does it work only with managed prometheus?

Reply:
Works with any Prometheus. Will add the benefit.